webapp/portlet/src/main/webapp/vue-apps/space-settings/components/SpaceSettings.vue

```diff
@@ -26,17 +30,19 @@ export default {
     // add external components
     const externalComponents = extensionRegistry.loadComponents('external-space').map(component => component.componentOptions.componentImpl);
     this.spaceExternalSettings.push(...externalComponents);
-
-    document.addEventListener('addSpaceSettingsExternalComponents', (event) => {
-      if (event && event.detail) {
-        this.spaceExternalSettings.push(event.detail.componentImpl);
-      }
-    });
-
     document.addEventListener('hideSettingsApps', () => this.displayed = true);
     document.addEventListener('showSettingsApps', () => this.displayed = true);
   },
   mounted() {
+    document.addEventListener('addSpaceSettingsExternalComponents', (event) => {
```

Comment:
instead of using this specific Event Listener, can you use the default extensionRegistry events, See https://github.com/Meeds-io/gatein-portal/blob/develop/web/eXoResources/src/main/webapp/javascript/eXo/core/extensionRegistry.js#L59

`component-${app}-${componentName}-updated`